include/path_tracking_pid/details/filtered_error_tracker.hpp

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include <path_tracking_pid/details/fifo_array.hpp>
+
+namespace path_tracking_pid::details
+{
+
+// Error tracker for the last 3 error and filtered error values.
+class FilteredErrorTracker
```

Comment:
Can we rename this class then to `SecondOrderLowpass`. This makes it really clear what's happening here

Reply:
@Rayman fixed!